CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.62 | [PR#2998](https://github.com/bbc/psammead/pull/2998) Remove duplicate `jest-styled-components` package from root. |
```

Comment:
Should the message be unused package rather than duplicate? (I couldn't find another entry for jest-styled-components in package.json)

Reply:
It was a duplicate in the sense that `psammead-test-helpers`, whose dependencies end up in the root package-lock, also depended on `jest-styled-components`. But I agree "unused" might make more sense here.